radixdlt/src/main/java/com/radixdlt/middleware2/network/TestEventCoordinatorNetwork.java

```diff
@@ -115,37 +111,40 @@ public void setReceivingDisable(ECPublicKey validatorId, boolean disable) {
 		}
 	}
 
-	private int getRandomLatency() {
-		if (minimumLatency == maximumLatency) {
-			return minimumLatency;
-		} else {
-			return minimumLatency + rng.nextInt(maximumLatency - minimumLatency + 1);
-		}
-	}
-
 	public EventCoordinatorNetworkSender getNetworkSender(ECPublicKey forNode) {
-		Consumer<MessageInTransit> sendMessageSink = message -> {
-			if (!sendingDisabled.contains(forNode)) {
-				if (preserveOrder) {
-					orderedMessageBuffer.push(message);
+		final Map<ECPublicKey, Long> lastTimestamps = new ConcurrentHashMap<>();
+
+		final Consumer<MessageInTransit> sendMessageSink = message -> {
```

Comment:
This seems pretty ordinary.  It looks like this is here just so we can include this method inside another method.

Really should be broken out into a separate method.